db/migrate/20230925123458_create_text_styles.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTextStyles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :text_styles do |t|
```

Comment:
Consider adding validations to ensure data integrity. For example, you might want to validate the presence of the 'name' attribute and ensure 'font_size' is a positive integer.